.travis.yml

```diff
@@ -29,6 +29,14 @@ matrix:
     - os: osx
       go: 1.11.x
       script:
+        - echo "Increase the maximum number of open file descriptors on macOS"
+        - NOFILE=20480
```

Comment:
Is this for "NO FILE" or "Number Of FILE" (with a final "S" missing) ? if it's the latter, call it `NOFILES` to avoid the confusion.

Reply:
@gballet no, don't revert changes to `mailserver.go`, I think it is fine as is.

Regarding `NOFILE` - I think this is how open file descriptors are called in Linux - for example https://wiki.archlinux.org/index.php/Limits.conf#nofile

Also it is pretty clear what this is doing based on the line above, and the commands below.

I don't have strong feelings about the name of the var, I can change to whatever if necessary.

Reply:
ok